### ISSUE TITLE
[BUG] UNSUPPORTED_URLS_FILE missing special characters from TITLE

### ISSUE THREAD

**Issue by the444xg** [Jules-WinnfieldX/CyberDropDownloader#611]
**Describe the bug**
The generated csv file removes special characters like "ã" from the TITLE field

**Setup Information**
 - OS: Ubuntu 22
 - Cyberdrop Downloader Version 4.2.218

**Comment by Jules-WinnfieldX** [comment]
Correct. This was done as some systems do not support these characters in directory names. Title is essentially used for these as is. 

I'll try and separate them in V5.

**Comment by the444xg** [comment]
Its useful mostly for matching names when organizing albums. Include thread ids is better for it since titles sometimes change or includes multiple names, i guess there's already a request for it.
Thanks for looking into it!
